src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -385,4 +386,19 @@ public Job call() throws SaltStackException {
         };
         return executor.submit(callable);
     }
+
+  public Stats stats() throws SaltStackException {
```

Comment:
So far all the methods have a sync and async implementation so i wouldn't make an exception here to keep the consistency.